userpanel/modules/helpdesk/style/bclean/templates/helpdesklist.html

```diff
@@ -1,18 +1,35 @@
 {box title="Request history"}
+
+<script type="text/javascript">
+	$(document).ready(function($) {
+		$(".table-click").click(function() {
+			window.document.location = $(this).data("href");
+		});
+	});
+</script>
+
+<style>
+	.table-click tr,td {
+		color: black;
+	}
+	.table-click tr :hover, td :hover {
+		cursor: pointer;
+		color: #007bff;
```

Comment:
O - widzę kolor na sztywno w szablonie html zaszyty - a jak potem temat graficzny zmienić? Szablony podmieniać? :)

Reply:
Czy jest dostępny jakikolwiek inny temat graficzny?;>

To co proponujesz - czy zrobić dodatkową klasę superblue z określonym kolorem?:-)

Reply:
nazywanie klasy superblue - chyba nie jest dobre, bo jak potem będzie chciał zmienić kolor na czerwony to będziesz miał czerwone superblue. Klasy raczej trzeba nazywać znaczeniowo (semantycznie), a nie bezpośrednio oddając ich wygląd. Zobacz: w CORE masz klasę hightlight i dla nie jest definiowany wewnątrz kolor.